### ISSUE TITLE
hCAPTCHA updates from fable test

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#5208]
Based on the result from the fable tests of the hCAPTCHA challenge (prompt). Note that these changes will not impact the current implementation that does not show a hCAPTCHA challenge. This is to prepare for further a11y testing. 

The following could be improved

Can Fix:
- [x] font colour (increase contrast) *sort of*
- [x] background and border (increase contrast)

And the following would require an hCAPTCHA support ticket:

Cannot fix:
- font size (increase size to 16/18)
- challenge button text not matching button accessible name
- challenge missing a close button (but does close on ESC and clicking outside challenge)
